jraph/_src/utils.py

```diff
@@ -15,14 +15,17 @@
 """Utilities for working with `GraphsTuple`s."""
 
 import functools
-from typing import Any, Callable, Iterable, List, Mapping, Optional, Sequence, Union
+from typing import Any, Callable, Iterable, List, Mapping, Optional, Sequence, \
+  Union
 
 import jax
 import jax.numpy as jnp
+import jax.ops
```

Comment:
I don't think this is needed, can't you directly call jax.ops.segment_sum etc.?

Reply:
You can, but jax/__init__.py doesn't explicitly import jax.ops, it looks like accident of dependency resolution that it's available so editors will give a warning.